eng/pipelines/templates/jobs/ci.tests.yml

```diff
@@ -216,7 +216,7 @@ jobs:
           test-proxy restore -a $(Build.SourcesDirectory)/sdk/${{parameters.ServiceDirectory}}
         workingDirectory: '$(Build.SourcesDirectory)/sdk/${{parameters.ServiceDirectory}}'
         displayName: Restore Recordings
-        condition: and(succeeded(), contains(variables.CmakeArgs, 'BUILD_TESTING=ON'), ne('${{parameters.ServiceDirectory}}', 'core'),ne('${{parameters.ServiceDirectory}}', 'template'))
+        condition: and(succeeded(), contains(variables.CmakeArgs, 'BUILD_TESTING=ON'), ne('${{parameters.ServiceDirectory}}', 'core'), ne('${{parameters.ServiceDirectory}}', 'template'), ne('${{parameters.ServiceDirectory}}', 'appconfiguration'))
```

Comment:
There must be a better pattern for these step conditionals than hardcoding a bunch of services. I'll do some thinking.

Reply:
@benbp -- The solutions exist in other languages today. I have an issue for this: https://github.com/Azure/azure-sdk-for-cpp/issues/6187

Reply:
FWIW, most service SDKs will have test recordings and hence won't need this opt-out.
So, I expect this list to include: core, template, and potentially a very nascent version of a service SDK (which will eventually be removed)

If we start getting several new C++ service SDKs spun up around the same time, then yes, this list could grow a bit more, before settling back down.